plugins/providers/gcs/client.go

```diff
@@ -0,0 +1,89 @@
+package gcs
+
+import (
+	"context"
+	"fmt"
+
+	"cloud.google.com/go/iam"
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+)
+
+//go:generate mockery --name=GcsClient --exported --with-expecter
+type GcsClient interface {
```

Comment:
```suggestion
type GCSClient interface {
```

reference https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Thanks for the reference mas @mabdh. Will update the changes accordingly